scripts/check_site.lua

```diff
@@ -289,6 +289,19 @@ function M.need_number(path, required)
 	return need_type(path, 'number', required, 'be a number')
 end
 
+function M.need_number_range(path, min, max, required)
```

Comment:
`need_number_range` has to be added to .luacheckrc for check_site.lua files.

Reply:
Ah, that makes sense. Thanks!